Libs/MRML/Core/vtkMRMLMarkupsNode.h

```diff
@@ -468,6 +468,9 @@ class  VTK_MRML_EXPORT vtkMRMLMarkupsNode : public vtkMRMLDisplayableNode
   void UnsetNthControlPointPosition(int pointIndex);
 
   /// Set control point status to ignored.
+  /// This method is excluded from wrapping to prevent automatic recognition as a `SET_NTH` function,
+  /// a behavior introduced in VTK 9.4 (kitware/VTK@5672af1f), which requires specific function signatures.
```

Comment:
```suggestion
  /// Excluded from wrapping to prevent unintended `SET_NTH` recognition.
  /// This behavior was introduced in VTK 9.4 (kitware/VTK@5672af1f) and requires
  /// specific function signatures. Until the VTK wrapping infrastructure is improved,
  /// this method should remain excluded. See https://gitlab.kitware.com/vtk/vtk/-/issues/19627.
```